### PR TITLE
fixed grid usage in content form

### PR DESCRIPTION
**Pull request by danrot**
| Q | A
| --- | ---
| Bug fix? | yes
| New feature? | no
| BC breaks? | no
| Deprecations? | no
| Fixed tickets | ---
| Related issues/PRs | ---
| License | MIT
| Documentation PR | ---

#### What's in this PR?

This PR reduces the code duplication in the twig templates for the content form.

#### Why?

Because the code was duplicated with errors. In the section the `div`s with the class `row` were not added, so the grid had errors when it was used within a section.

To test the following code can be put into a template:

```xml
        <section name="test">
            <properties>
                <property name="test1" type="text_line" colspan="6"></property>
                <property name="test2" type="text_line" colspan="6"></property>
                <property name="test3" type="text_line"></property>
                <property name="test4" type="text_line" colspan="6"></property>
                <property name="test5" type="text_line" colspan="6"></property>
            </properties>
        </section>
```

### CODE REVIEW

File: src/Sulu/Bundle/ContentBundle/Resources/views/Template/content-types/block.html.twig [hunk 1]
```diff
@@ -51,36 +51,10 @@
...
-                                        {% set columns = childColspan %}
-                                    {% endif %}
-
-                                    {# get Type for property #}
-                                    {% set type = sulu_get_type(child.contentTypeName) %}
-                                    {# get params for property #}
-                                    {% set params = sulu_get_params(child) %}
-
-                                    {% if child is multiple %}
-                                        {% include 'SuluContentBundle:Template:macros/multiple.html.twig' with {'property': child, 'params': params, 'type': type, 'id': id ~ '-' ~ child.name ~ '<~=index~>', 'webspaceKey': webspaceKey, 'languageCode': languageCode, 'userLocale': userLocale} only %}
```
Comment (danrot) [inline src/Sulu/Bundle/ContentBundle/Resources/views/Template/content-types/block.html.twig]: I think I have to readd the `<~=index~>` somehow. Although I have left them out for now, and everything seems to be still working. @wachterjohannes Any idea why this part is needed?

Comment (danrot) [inline src/Sulu/Bundle/ContentBundle/Resources/views/Template/content-types/block.html.twig]: It caused problems when a block contained a property which was already existing outside with the same name. Added it again.

File: src/Sulu/Bundle/ContentBundle/Resources/views/Template/macros/properties.html.twig [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+        {% set columns = propertyColspan %}
+    {% endif %}
+
+    {# get params for property #}
+    {% set params = sulu_get_params(property) %}
+    {% set id = propertyPrefix|default('') ~ property.name ~ propertyPostfix|default('') %}
+
+    {% if property.contentTypeName == 'section' %}
+        {% include 'SuluContentBundle:Template:macros/section.html.twig' with {
+        'property': property,
```
Comment (wachterjohannes) [inline src/Sulu/Bundle/ContentBundle/Resources/views/Template/macros/properties.html.twig]: this format is quite hard to read shouldnt we intend this parameters with 4 spaces? i know intellij will remove the always but for readability i would do it.

**Comment by wachterjohannes** [comment]
@danrot functionally everything works

**Comment by danrot** [comment]
@wachterjohannes fixed

**Pull request merged**